bootstrap_cfn/fab_tasks.py

```diff
@@ -317,3 +319,26 @@ def rsync():
         remote_dir=remote_pillar_dir,
         local_dir=os.path.join(local_pillar_dir, env.environment, '.'),
         use_sudo=True)
+
+@task 
+def update_certs():
+    """
+    Update the ssl certificates with those in the config file.
+    Also handle settings the certificates on ELB's
+    """
+    stack_name = get_stack_name()
+    aws_config, cfn, cfn_config = get_config()
+    # Upload any SSL certificates to our EC2 instances
+    if 'ssl' in cfn_config.data:
+        iam = IAM(aws_config)
+        iam.delete_ssl_certificate(cfn_config.ssl(), stack_name)
+        iam.upload_ssl_certificate(cfn_config.ssl(), stack_name)
+    # Arbitrary wait to allow SSL upload to register with AWS
+    # Otherwise, we can get an ARN for the load balancer certificates
+    # without it being ready to assign
+    time.sleep(3)
+
+    # Set the certificates on ELB's
+    if 'elb' in cfn_config.data:
+        elb = ELB(aws_config)
+        elb.set_ssl_certificates(cfn_config.ssl(), stack_name)
```

Comment:
Similar thing about the `if`s here - we should check the both exist to start with and error if they don't.